lib/shopify_app/shop.rb

```diff
@@ -11,5 +11,15 @@ def with_shopify_session(&block)
       ShopifyAPI::Session.temp(shopify_domain, shopify_token, &block)
     end
 
+    def connect
+      ShopifyAPI::Base.activate_session(
+        ShopifyAPI::Session.new(
+          ShopifyApp::Utils.sanitize_shop_domain(shopify_domain),
+          shopify_token
+        )
+      )
+      nil
+    end
+
```

Comment:
I would prefer it written like so to avoid extra nesting:

```ruby
def connect
  session = ShopifyAPI::Session.new(shopify_domain, shopify_token)
  ShopifyAPI::Base.activate_session(session)
end
```

I'd also drop the nil return.